### ISSUE TITLE
SSL support for Mariadb columnstore destination connector

### ISSUE THREAD

**Issue by animer3009** [airbytehq/airbyte#15173]
Hi guys,
Can I ask to add ssl support for for Mariadb columnstore destination connector?

We are using SkySQL cloud Mariadb columnstore cloud database and they do not support connection without ssl.

**Comment by natalyjazzviolin** [comment]
@animer3009 do you have any ideas for a solution or an alternative you're currently using/have considered using? I've edited the initial comment with our standard template - please feel free to fill it out some more if you've got more details/ideas!

**Comment by animer3009** [comment]
@natalyjazzviolin 
Nope, we just asking for SSL support for Mariadb CS destination connector

**Comment by natalyjazzviolin** [comment]
Got it! I've triaged this to the appropriate team, so they should take it over from here!

**Comment by yurii-bidiuk** [comment]
MariaDB supports the following SSL modes:
**disable**: Do not use SSL/TLS (default)
**trust**: Only use SSL/TLS for encryption. Do not perform certificate or hostname verification. This mode is not safe for production applications.
**verify-ca**: Use SSL/TLS for encryption and perform certificates verification, but do not perform hostname verification.
**verify-full**: Use SSL/TLS for encryption, certificate verification, and hostname verification

In the scope of this ticket we need to:
- Modify spec.json by adding a drop-down menu with SSL modes described above.
- Implement handling for each mode and certificates generation on the destination side
- Implement integration tests
- Configure the test MariaDB container to enable SSL for the server
- Verify if normalization requires any changes for SSL

**Comment by octavia-squidington-iii** [comment]
At Airbyte, we seek to be clear about the project priorities and roadmap. This issue has not had any activity for 365 days, suggesting that it's not as critical as others. It's possible it has already been fixed. It is being marked as stale and will be closed in 20 days if there is no activity. To keep it open, please comment to let us know why it is important to you and if it is still reproducible on recent versions of Airbyte.
